src/main/java/com/swervedrivespecialties/swervelib/SwerveModuleFactory.java

```diff
@@ -90,5 +90,14 @@ public void set(double driveVoltage, double steerAngle) {
             driveController.setReferenceVoltage(driveVoltage);
             steerController.setReferenceAngle(steerAngle);
         }
+
+        @Override
+        public double getAbsoluteAngle() {
+            if (steerController instanceof AbsoluteEncoder) {
+                AbsoluteEncoder encoder = (AbsoluteEncoder)steerController;
+                return encoder.getAbsoluteAngle();
+            }
+            return Double.NaN;
```

Comment:
There might be some future bugs if code uses NaN values, but they shouldn't be too hard to fix. We may want to mention this in a doc comment, but that depends on who we expect to use this.